src/components/missions/mission.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { Button } from 'react-bootstrap';
+import { useDispatch } from 'react-redux';
+import { JoinMission, LeaveMission } from '../../redux/mission/missionActions';
+
+function Mission({
+  // eslint-disable-next-line react/prop-types
+  id, description, mission, status,
+}) {
+  const dispatch = useDispatch();
+  return (
+    <tr>
+      <td>{mission}</td>
+      <td>{description}</td>
+      <td>
+        {
+          status
+            ? (
+              <p className="bg-success text-white">
+                Active Member
+              </p>
+            )
+            : (
+              <p className="bg-secondary text-white">
+                Not a member
+              </p>
+            )
+    }
```

Comment:
[OPTIONAL] This block of code might be improved a bit to conform DRY principle as such:
```
<p className={(status ? 'bg-secondary' : 'bg-success') + ' text-white'}>
          {status ? 'Active Member' : 'Not a member'}
</p>
```